Tests/XCUITests/PhotonActionSheetTest.swift

```diff
@@ -101,14 +101,15 @@ class PhotonActionSheetTest: BaseTestCase {
 
         // This is not ideal but only way to get the element on iPhone 8
         // for iPhone 11, that would be boundBy: 2
+        waitForExistence(app.collectionViews.buttons["Copy"], timeout: TIMEOUT)
+        waitForExistence(app.collectionViews.scrollViews.cells["XCElementSnapshotPrivilegedValuePlaceholder"].firstMatch, timeout: TIMEOUT)
```

Comment:
For both iPhone and iPad, let me ensure that the parts from the `collectionViews` is loaded first just to ensure `fennecElement` is the one we want.

I have run the test repeatedly (10 times) locally with this change on both iPhone 14 and iPad.